Source/TestDafny/MultiBackendTest.cs

```diff
@@ -368,7 +442,8 @@ int ExpectedExitCode
       var checkDirectives = OutputCheckCommand.ParseCheckFile(checkFile);
       var checkResult = OutputCheckCommand.Execute(errorWriter, outputLines, checkDirectives);
       if (checkResult != 0) {
-        await output.WriteLineAsync($"OutputCheck on {checkFile} failed:");
+        await output.WriteLineAsync($"OutputCheck on {checkFile} failed. Output was:");
+        await output.WriteLineAsync(string.Join("\n", outputLines.Take(5)));
```

Comment:
I would not truncate the output, especially not without communicating that. I've found Rider at least handles tests with a lot of output well (it truncates it itself with a link to view the whole output)